app/mobile/src/components/atoms/LargeButton.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { StyleSheet, TouchableOpacity, Text } from "react-native";
+import { COLORS } from "../../constants/colorPalette";
+
+interface ButtonProps {
+  title: string;
+  onPress: any;
+  theme?: "primary" | "secondary" | "accent";
+  height?: number | string;
+  width?: number | string;
+  fontSize?: number;
+  color?: string;
+}
+
+export default function CustomButton({
```

Comment:
```suggestion
export default function LargeButton({
```
We try to make the component name match the file name for consistency and less confusion.